test/helper/Appium_test.js

```diff
@@ -24,8 +24,8 @@ describe('Appium', function () {
     }
 
     app = new Appium({
+      app: apk_path,
```

Comment:
But maybe let's keep it there?
Putting it to master will make it the part of the package whic is not a good idea, and I don't know any other free reliable hosting other than GitHub